.github/workflows/lint.yml

```diff
@@ -7,6 +7,7 @@ on:
       - master
       - main
       - release/*
+      - landchecks/*
```

Comment:
I think we should add this to everything we can revert for--let's no longer have the confusion of "what is currently considered a land time check?". So trunk + pull + lint + the other vestiges that one can be reverted for should all have this.

Reply:
I thought we wanted to just start with lint so the tts isn't too bad/for testing purposes?

Reply:
Agreed with @janeyx99 - we should not land a PR that introduces landchecks without actually waiting for trunk.yml to finish